pyqtgraph/examples/ExampleApp.py

```diff
@@ -352,6 +360,27 @@ def onComboChanged(searchType):
         self.ui.exampleTree.itemDoubleClicked.connect(self.loadFile)
         self.ui.codeView.textChanged.connect(self.onTextChange)
         self.codeBtn.clicked.connect(self.runEditedCode)
+        self.updateCodeViewTabWidth(self.ui.codeView.font())
+
+    def updateCodeViewTabWidth(self,font):
+        """
+        Change the codeView tabStopDistance to 4 spaces based on the size of the current font
+        """
+        fm = QtGui.QFontMetrics(font)
+        tabWidth = fm.horizontalAdvance(' ' * 4)
+        # the default value is 80 pixels! that's more than 2x what we want.
+        self.ui.codeView.setTabStopDistance(tabWidth)
+
+    def showEvent(self, event) -> None:
+        super(ExampleLoader, self).showEvent(event)
+        disabledColor = pg.mkColor(QtCore.Qt.GlobalColor.red)
```

Comment:
may I suggest that this PR not change the public API at the same time?
`QtGui.QColor(QtCore.Qt.GlobalColor.red)` would seem to suffice.